### PR TITLE
Version 1.2

### PR DESCRIPTION
**Pull request by bakennedy**
Version increase for event buffer max send time

### CODE REVIEW

File: version.sbt [hunk 1]
```diff
@@ -1 +1 @@
-ThisBuild / version := "1.11"
\ No newline at end of file
+ThisBuild / version := "1.2"
```
Comment (praves77) [inline version.sbt:1]: I don't know what's convention we're using. Should it be 1.20? 
Otherwise it looks fine.

Comment (praves77) [reply version.sbt:1]: @bakennedy ^^^

Comment (keyur9) [reply version.sbt]: Also, @bakennedy we need to update the readme with the new version as well.

**Review by praves77 — APPROVED** [review]
lgtm

File: version.sbt (hunk 1 above)
Comment (praves77) [reply version.sbt]: Good catch @keyur9

**Review by dgilling — APPROVED** [review]
Let me know if want to walk through publishing. I believe @CodeYouMust , set up the builds in Azure DevOps we can trigger to publish to maven

**Comment by praves77** [comment]
> Let me know if want to walk through publishing. I believe @CodeYouMust , set up the builds in Azure DevOps we can trigger to publish to maven

Would love to see it. Much appreciated. @dgilling or @CodeYouMust whoever has time can walk through publishing.

**Comment by CodeYouMust** [comment]
@bakennedy So the publish to mavencentral is very very simple: In azure devops, Just click on button ["Create new release" on job `Publish SDK moesif-play-filter`](https://dev.azure.com/Moesif/Website/_release?_a=releases&view=all&definitionId=16)
In short it does the following:
pulls repo, builds, tests, PGP signs it, packages and publishes it to maven. It pushes it thru the maven central lifecycles all using the `sbt` command.

The only issue is, that because I set this up, the `Azure devops` <--to--> `github` pipeline seems to be using my git credentials, so that step fails for anyone other than me. 

To fix this issue, we need to figure out how git should connect with azure devops. I opened a ticket here: https://www.pivotaltracker.com/story/show/180628543 @praves77 can help triage.

I just executed the above for this PR `v1.2` and [it successfully published it - logs](https://dev.azure.com/Moesif/Website/_releaseProgress?_a=release-environment-logs&releaseId=6168&environmentId=7224). While we have successfully published it, do note that maven repo search/index can take a few minutes to several hours. during the log4j fiasco, there were failures and delays.

**Pull request merged**